selenium_tests/course_info/page_objects/course_info_detail_page_object.py

```diff
@@ -36,12 +47,29 @@ def get_input_field_value(self, field_name):
         input = self.find_element(*Locators.INPUT_BY_FIELD_NAME(field_name))
         return input.get_attribute('value')
 
+    def is_element_displayed_as_input_field(self, field_name):
+        is_displayed = self.find_element(*Locators.INPUT_BY_FIELD_NAME(
```

Comment:
This will throw a `NoSuchElementException` if not found, instead of returning `False`, right? Do you want to wrap it in a try-except block?